.github/workflows/ci.yml

```diff
@@ -0,0 +1,105 @@
+name: 'CI'
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  Checks:
+    runs-on: ubuntu-18.04
```

Comment:
Definitely not a blocker, but curious if we are able to use ubuntu-20.04?

Reply:
Issues with Open SSL 1.1.0 and Crypto. Added card here: https://trello.com/c/v9p7Shgr/12-upgrade-ci-to-use-ubuntu-2004

Reply:
🎉